requirements/requirements-dev.txt

```diff
@@ -15,5 +15,5 @@ sphinx
 sphinx-rtd-theme
 tensorboard
 torchvision
-transformers>=4.32.1
+transformers @ git+https://github.com/loadams/transformers@loadams/fix-pytest-import
```

Comment:
This is just for testing, will remove this when the changes in that branch are merged into transformers.

Reply:
Merged into transformers - need 4.39.0-dev.0 minimum.